rm-datasource/src/main/java/io/seata/rm/datasource/xa/ResourceManagerXA.java

```diff
@@ -75,9 +75,10 @@ public void initXaTwoPhaseTimeoutChecker() {
                         xaTwoPhaseTimeoutChecker.scheduleAtFixedRate(() -> {
                             for (Map.Entry<String, Resource> entry : dataSourceCache.entrySet()) {
                                 BaseDataSourceResource resource = (BaseDataSourceResource)entry.getValue();
-                                if (resource.isShouldBeHeld()) {
-                                    if (resource instanceof DataSourceProxyXA) {
-                                        Map<String, ConnectionProxyXA> keeper = resource.getKeeper();
+                                if (resource instanceof DataSourceProxyXA) {
+                                    BaseDataSourceResourceXA<ConnectionProxyXA> resourceXA = (BaseDataSourceResourceXA<ConnectionProxyXA>) resource;
+                                    if (resourceXA.isShouldBeHeld()) {
```

Comment:
What does it have to do with here?

https://github.com/seata/seata/pull/5034/files#diff-f01c43d21a35df90364bd1a6ae03c003ef1d9a99e4938a392bc2b64583605f27R72

Reply:
The variable name `shouldBeHold` should be changed to clear meaning.

Reply:
done